datalad/interface/run_procedure.py

```diff
@@ -451,6 +451,7 @@ def __call__(
         lgr.debug('Attempt to run procedure {} as: {}'.format(
             name,
             cmd))
+        lgr.info("Running procedure %s", name)
```

Comment:
Maybe "upgrade" the debug message on the line before? Two log messages with almost identical content seem unnecessary.

Reply:
Indeed! Good catch - I was blind.  Making it 
```python
        lgr.info('Running procedure %s', name)
        lgr.debug('Procedure command is %r', cmd)
```
this way we can see gory details of the command for debugging purposes (like apparently there is some trailing space ATM, does not matter but might come critical, who knows).

BTW looking at the 
```python
        lgr.debug('Attempt to run procedure {} as: {}'.format(
            name,
            cmd))
```
it is imho better to delay any string formatting until (and if) actually needed, that is why I use `%` form, and would do
```python
        lgr.debug('Attempt to run procedure %s as: %s', name, cmd)
```
I didn't time the effects, but assume that it should spare a cycle or two ;-)

Reply:
I pushed such a modification.